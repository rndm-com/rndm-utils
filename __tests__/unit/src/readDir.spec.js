const readDir = require('../../../src/readDir');
const { describe, test } = global;

describe('src/readDir', () => {
  test('current', () => {
    const input = __dirname;
    const sut = readDir(__dirname).filter(i => !i.includes('__snapshots__')).map(i => i.replace(process.cwd(), ''));
    expect(sut).toMatchSnapshot()
  });
});
