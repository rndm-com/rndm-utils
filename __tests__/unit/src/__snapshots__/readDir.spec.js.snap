// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/readDir current 1`] = `
Array [
  "/__tests__/unit/src/buildFilesForFolder.spec.js",
  "/__tests__/unit/src/decircular.spec.js",
  "/__tests__/unit/src/index.spec.js",
  "/__tests__/unit/src/mkDir.spec.js",
  "/__tests__/unit/src/readDir.spec.js",
]
`;
